CSS-10880 Improve documentation (#53)
canonical/superset-k8s-operator: documentation/reference/revision-resource-compatibility.md

# Compatible Charm Revisions and Resources

Deploying charms from the `latest/stable` channel *should* guarantee 
compatibility between charm revision and resource.

> However, if you are deploying earlier revisions of the charm please

Reviewer: "Revisions" or "versions"? I am not sure about the technical term in this case

Author: `revisions` = the revision of the charm code, `version` = the applications version. So I think revision is the correct use here

> However, if you are deploying earlier revisions of the charm please

Reviewer: ```suggestion
However, if you are deploying earlier revisions of the charm, 
```

> use the below table as a reference for compatibility.

Reviewer: ```suggestion
use the table below as a reference for compatibility:
```

> | Revision | Resource revision | Resource version |

Reviewer 2: praise: I like this table.

|----------|-------------------|------------------|
| 34       | 18                | 3.1.3            |
| 33       | 18                | 3.1.3            |
| 32       | 15                | 3.1.3            |
| 31       | 14                | 2.1.0            |

> A revision and resource version can be specified in the deployment:

Reviewer: ```suggestion
You can specify a revision and resource version can in the deployment as follows:
```
In how-to's we tend to use second person for actions the user can do.

Author: Got it thank you!
